src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,55 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import mate.academy.lib.Inject;
+
+@Entity
+@Inject
+@Table(name = "cinema Hall")

```

Comment:
The `@Table` annotation should not contain any spaces. Please rename it to `cinema_hall` or `cinemaHall` to follow the naming convention.